### PR TITLE
prepare application for developer hub

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,165 @@
...
+| Integrations | CDK, AWS CLI                                                                            |
+| Categories   | Serverless; Event-Driven architecture                                                   |
+| Level        | Intermediate                                                                            |
+| GitHub       | [Repository link](https://github.com/localstack/loan-broker-stepfunctions-lambda-app)   |
+
+## Introduction
+
+The Loan Broker application with AWS Step Functions, DynamoDB, Lambda, SQS, and SNS demonstrates Gregor Hohpe's [Loan Broker example](https://www.enterpriseintegrationpatterns.com/ramblings/loanbroker_stepfunctions.html). The sample application implements a [Recipient List](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RecipientList.html) pattern and a [Scatter Gather](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RecipientList.html) pattern to retrieve a list of banks and dynamically route loan application to multiple banks respectively. The sample application implements the following integration among the various AWS services:
```
Comment (steffyP) [inline README.md]: The link for `Scatter Gather` also links to the `RecipientList`, I think you meant to link this instead: https://www.enterpriseintegrationpatterns.com/patterns/messaging/BroadcastAggregate.html

File: README.md [hunk 2]
```diff
@@ -1 +1,165 @@
...
+```shell
+export LOCALSTACK_API_KEY=<your-api-key>
+DEBUG=1 localstack start
+```
+
+We specified DEBUG=1 to get the printed LocalStack logs directly in the terminal to help us see the event-driven architecture in action. If you prefer running LocalStack in detached mode, you can add the `-d` flag to the `localstack start` command, and use Docker Desktop to view the logs.
+
+## Instructions
+
+You can build and deploy the sample application on LocalStack by running our `Makefile` commands. To deploy the infrastructure, you can run `make deploy` and `make run` after installing the application dependencies. Here are instructions to deploy and test it manually step-by-step.
```
Comment (steffyP) [inline README.md]: I don't see a `Makefile` in the PR - did you forget to add it, or is it a copy-paste error?

File: README.md [hunk 3]
```diff
@@ -1 +1,165 @@
...
+
+### Deploying the application
+
+To create the AWS infrastructure locally, you can use CDK and our `cdklocal` wrapper. Before you can deploy the infrastructure, you need to install the application dependencies:
+
+```sh
+yarn
```
Comment (steffyP) [inline README.md:60]: we could also list `yarn` in the Prerequisites section

File: diagram.png
Comment (steffyP) [inline diagram.png:1]: did you intended to add this diagram to the `README`? It seems to be missing

File: README.md [hunk 4]
```diff
@@ -1 +1,165 @@
...
+
+Before you can test the application, you need to pre-populate the `LoanBrokerBanksTable` for the `RecipientsList` stack:
+
+```sh
+awslocal dynamodb put-item \
+    --table-name=LoanBrokerBanksTable \
+    --item='{ "Type": { "S": "Home" }, "BankAddress": {"L": [ { "S": "BankRecipientPremium" }, { "S": "BankRecipientUniversal" }, { "S": "BankRecipientPawnshop" } ] } }'
+```
+
+Start the State Machine execution by running the following command:
```
Comment (steffyP) [inline README.md]: could we add some more explanation here, e.g. outline that this is how we can get quotes from the banks/submit a the loan application?

File: README.md [hunk 5]
```diff
@@ -1 +1,165 @@
...
+
+You can convert your current AWS infrastructure state to a Cloud Pod using the `localstack` CLI. Check out our [Getting Started guide](https://docs.localstack.cloud/user-guide/tools/cloud-pods/getting-started/) and [LocalStack Cloud Pods CLI reference](https://docs.localstack.cloud/user-guide/tools/cloud-pods/pods-cli/) to learn more about Cloud Pods and how to use them.
+
+To inject a Cloud Pod that contains the infrastructure for this application, you can run the following command:
+
+```sh
+localstack pod load file://$(pwd)/cloud-pod/loan-broker-application
+```
+
+Alternatively, you can use [Cloud Pods Launchpad](https://docs.localstack.cloud/user-guide/tools/cloud-pods/launchpad/) to quickly inject Cloud Pods into your running LocalStack container. Click on the [badge] to launch Cloud Pods Launchpad and inject the Cloud Pod for this application by clicking the `Inject` button.
```
Comment (steffyP) [inline README.md:176]: Do you plan to add the cloudpod badge here already, or should we remove this section for now?

**Review by steffyP — COMMENTED** [review]
Amazing work @HarshCasper 🎉 👍 Thanks for adding your hackathon project so quickly 🥳 

I don't have too much knowledge with cdk and js, so my review focuses on the README, explanation and setup of the sample.

File: diagram.png
Comment (HarshCasper) [reply diagram.png:1]: Yes, it needs to be completed

File: README.md (hunk 5 above)
Comment (HarshCasper) [reply README.md:176]: I plan to do that once we merge this.

File: README.md [hunk 6]
```diff
@@ -1 +1,166 @@
...
+
+Before you can test the application, you need to pre-populate the `LoanBrokerBanksTable` for the `RecipientsList` stack:
+
+```sh
+awslocal dynamodb put-item \
+    --table-name=LoanBrokerBanksTable \
+    --item='{ "Type": { "S": "Home" }, "BankAddress": {"L": [ { "S": "BankRecipientPremium" }, { "S": "BankRecipientUniversal" }, { "S": "BankRecipientPawnshop" } ] } }'
+```
+
+We can start the State Machine execution to get quotes from the banks after submittin a loan application. Run the following command:
```
Comment (steffyP) [inline README.md]:
```suggestion
We can start the State Machine execution to get quotes from the banks after submitting a loan application. Run the following command:
```

**Review by steffyP — APPROVED** [review]
Thanks for addressing and clarifying my comments @HarshCasper 🙏 
Everything LGTM, I just noted a small typo. Great work! 🎉

**Pull request merged**